.github/workflows/pr.yml

```diff
@@ -0,0 +1,21 @@
+name: PR
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  workflow_call:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  run_g_tests:
+    name: Run LabVIEW Tests
+    if: ${{ github.repository == 'ni/measurementlink-labview' || github.event.pull_request.head.repo.full_name == 'ni/measurementlink-labview' }}
+    uses: ./.github/workflows/run_g_tests.yml
+
+  run_pr_commenter:
+    name: Run PR Commenter
+    if: ${{contains(github.event.pull_request.body, 'RUN_G_DIFF_CHECK') && (github.repository == 'ni/measurementlink-labview' || github.event.pull_request.head.repo.full_name == 'ni/measurementlink-labview') }}
```

Comment:
I think having this opt-in is a good idea and we might consider leaving it opt-in permanently. It isn't that uncommon for us to make a change that touches hundreds of VIs (hopefully that will be less common going forward). In that case, I don't think I'd want this tool to run. Maybe we can add a comment to the PR template like this:

`<!-- RUN_G_DIFF_CHECK: Uncomment this line to enable VI diffing. Consider leaving VI diffing disable for PRs that modify a lot of VIs  -->`

And then you can modify the logic in `pr.yml` to decide whether to run the differ. This seems like the best of all worlds.

Reply:
It might be tricky to check whether text is commented out. I don't know if `github.event.pull_request.body` strips out commented out text.

Consider using a checkbox like Renovate does: https://github.com/ni/measurementlink-labview/pull/407

markdown:
` - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box`

rendered:
 - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box

Reply:
When you check/uncheck a checkbox, GitHub automatically edits the PR body, reply, or whatever.

Reply:
Yeah, the checkbox is a good idea.

Reply:
Tracking this with a follow-up task: 2582143